renderer/components/Request/Request.js

```diff
@@ -156,11 +164,33 @@ class Request extends React.Component {
     )
   }
 
+  renderRoutingHints = () => (
+    <Flex alignItems="center" justifyContent="space-between">
+      <Flex>
+        <Span color="gray" fontSize="s" mr={2}>
+          <Padlock />
+        </Span>
+        <Flex>
+          <Label htmlFor="routingHints">
+            <FormattedMessage {...messages.routing_hints_label} />
+          </Label>
+          <Tooltip ml={1}>
+            <FormattedMessage {...messages.routing_hints_tooltip} />
+          </Tooltip>
+        </Flex>
+      </Flex>
+      <Flex>
```

Comment:
This can probably be a regular `Box` since only 1 child inside

Reply:
No need fo Box even, have removed completely.